### PR TITLE
Enabling Incrementals

### PR DESCRIPTION
**Pull request by bgermann**
### Submitter checklist

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -42,12 +42,15 @@
...
-        <url>https://github.com/jenkinsci/libvirt-slave-plugin</url>
-        <tag>HEAD</tag>
+        <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+        <url>https://github.com/${gitHubRepo}</url>
+        <tag>${scmTag}</tag>
     </scm>
     <properties>
+        <revision>1.9.3</revision>
+        <changelist>-SNAPSHOT</changelist>
```
Comment (NotMyFault) [inline pom.xml:52]: Should be `<changelist>999999-SNAPSHOT</changelist>` and remove the revision clause.

File: pom.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
     </parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>libvirt-slave</artifactId>
-    <version>1.9.3-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
```
Comment (NotMyFault) [inline pom.xml:12]: Should be `<version>${changelist}</version>`

**Review by NotMyFault — COMMENTED** [review]
- `-Dchangelist.format=%d.v%s` is missing in `.mvn/maven.config`
- `.github/workflows/cd.yaml` with the content of https://github.com/jenkinsci/console-column-plugin/blob/master/.github/workflows/cd.yaml is missing
- `.github/release-drafter.yml` with the content of https://github.com/jenkinsci/console-column-plugin/blob/master/.github/release-drafter.yml is missing

Why was this PR merged before https://github.com/jenkins-infra/repository-permissions-updater/pull/3343 was submitted? An open PR would have facilitated a fast review without the need to create a new one.

File: pom.xml (hunk 1 above)
Comment (bgermann) [reply pom.xml:52]: Okay, have created #29

**Pull request merged**